shub/config.py

```diff
@@ -35,9 +39,11 @@ def load(self, stream):
             yaml_cfg = yaml.safe_load(stream)
             if not yaml_cfg:
                 return
-            for option in ('projects', 'endpoints', 'apikeys'):
+            for option in ('projects', 'endpoints', 'apikeys', 'stacks'):
                 getattr(self, option).update(yaml_cfg.get(option, {}))
             self.version = yaml_cfg.get('version', self.version)
+            if 'requirements_file' in yaml_cfg:
+                self.requirements_file = yaml_cfg['requirements_file']
```

Comment:
and it seems like you can have one line here `self.requirements_file = yaml_cfg.get('requirements_file')`